RNReanimated.podspec

```diff
@@ -71,6 +71,7 @@ Pod::Spec.new do |s|
   s.dependency 'React-Core/DevSupport'
   if !config[:is_tvos_target]
     s.dependency 'React-RCTActionSheet'
+    s.dependency "React-hermes"
```

Comment:
What if someone uses JSC? We shouldn't declare a dependency on `React-hermes` in such case.